js/index.js

```diff
@@ -0,0 +1,61 @@
+let books = [];
+
+function searchID(id) {
+  return document.getElementById(id);
+}
+
+function showBook(object) {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.